services/ecosystems/src/persistence/coverageDryForest.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (db, { coverageDryForest }, logger) => ({
+  /**
+   * Find the area distribution for each coverage type in SE dry forest in a given area
+   *
+   * @param {String} areaType area type
+   * @param {String | Number} areaId area id
+   * @param {Number} year optional year to filter data, 2018 by default
+   *
+   * @returns {Object[]} Values of area distribution for each coverage type
+   */
+  findCoverageSEDryForest: (areaType, areaId, year = 2018) =>
```

Comment:
since it's now in the coverageDryforest persistence is redundant this name, I suggest `findCoverageDistribution` or something similar, that express a little more the purpose of the function.

> This applies for all 3 ecosystems

Reply:
Done!